editor/src/core/model/element-metadata-utils.ts

```diff
@@ -828,6 +828,20 @@ export const MetadataUtils = {
       }, Utils.asLocal(frame))
     }
   },
+  getFrameRelativeToTargetContainingBlock: function (
+    targetParent: ElementPath | null,
+    metadata: ElementInstanceMetadataMap,
+    frame: CanvasRectangle,
+  ): LocalRectangle {
+    const parent = this.findElementByElementPath(metadata, targetParent)
+    if (parent != null && parent.specialSizeMeasurements.coordinateSystemBounds != null) {
```

Comment:
is `coordinateSystemBounds` the right measurement to use here? that's the bounds of `targetParent`'s containing block!

Reply:
yes, this needs to use the globalFrame of the absolute parent. I'm adding a test for this.